### PR TITLE
REF: FlatList Wallet Carousel #3273

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/m4wn7773udxryh6fgxgb4epz20?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/FlatList-Wallet-Carousel-#3273-1623824782.apk)

File: package.json [hunk 1]
```diff
@@ -159,7 +159,6 @@
     "react-native-screens": "2.18.1",
     "react-native-secure-key-store": "https://github.com/BlueWallet/react-native-secure-key-store#63ab38c9d382a819844a086a69cc204c46aa93f9",
     "react-native-share": "6.1.0",
-    "react-native-snap-carousel": "3.9.1",
```
Comment (Overtorment) [inline package.json:162]: banderas tenor

**Review by Overtorment — APPROVED** [review]
lgtm. also, tested on appetize a bit, with ~6 wallets worked well

File: package.json (hunk 1 above)
Comment (marcosrdz) [reply package.json:162]: You'll be happy to know that another package is gonna be removed soon. WIP.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/54gfwy5096jf7m0tfn5edk3wjg?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/FlatList-Wallet-Carousel-#3273-1623940957.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/07288ww3hrhwfb98d41uzkgdr0?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/FlatList-Wallet-Carousel-#3273-1624033368.apk)

**Comment by Overtorment** [comment]
conflict

**Comment by marcosrdz** [comment]
> conflict

resolved

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/dc6bzr0md2fdnx4rhwjp5j2ufm?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/FlatList-Wallet-Carousel-#3273-1624380018.apk)

**Review by ncoelho — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/h15vybuwg0hn434y8keqp7v4rr?device=pixel4

[download apk](http://206.189.62.190:1488/FlatList-Wallet-Carousel-#3273-1624549605.apk)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**